editor/src/missing-types/index.d.ts

```diff
@@ -88,6 +88,8 @@ declare module 'friendly-words' {
 declare module 'react-windowed-select' {
```

Comment:
We already had this? Are there correct types for this available now?

Reply:
I could actually remove this